C/CSPICE/build_tarballs.jl

```diff
@@ -3,13 +3,13 @@
 using BinaryBuilder, Pkg
 
 name = "CSPICE"
-version = v"66.1.0"
+version = v"67.0.0"
 
 # Collection of sources required to complete build
 sources = [
     ArchiveSource(
-        "http://naif.jpl.nasa.gov/pub/naif/toolkit//C/PC_Linux_GCC_64bit/packages/cspice.tar.Z",
-        "93cd4fbce5818f8b7fecf3914c5756b8d41fd5bdaaeac1f4037b5a5410bc4768",
+        "http://naif.jpl.nasa.gov/pub/naif/toolkit/C/PC_Linux_GCC_64bit/packages/cspice.tar.Z",
```

Comment:
Are you saying they change the tarball in-place? :facepalm:

Reply:
They absolutely do 🤯